doc/spies/eventbridge.md

```diff
@@ -16,21 +16,21 @@ This helper creates a new spy for a given event bus. Parameters:
 
 Config:
 
-- `matcherDefaultTimeout`: The default timeout for event matchers, in milliseconds. Defaults to `10_000`. This is the maximum time a matcher will wait until it determines whether the assertion succeeds or fails. Also see the [recommendations](#recommendations) below about timeouts.
+- `matcherDefaultTimeout`: The default timeout for event matchers, in milliseconds. Defaults to `10000`. This is the maximum time a matcher will wait until it determines whether the assertion succeeds or fails. Also see the [recommendations](#recommendations) below about timeouts.
```

Comment:
I actually made this one on purpose for readability.

Reply:
I think it may make sense, however I saw other numbers didn't have this convention so I thought it was a typo as well 😅

Reply:
Yeah, we could make this consistent. 
I'm fine either way.

Reply:
I probably thought that 3 `0` was readable but after 4 it's harder.